src/abstracts/SablierV2Lockup.sol

```diff
@@ -199,6 +200,11 @@ abstract contract SablierV2Lockup is
         streamedAmount = _streamedAmountOf(streamId);
     }
 
+    /// @inheritdoc ERC721
+    function supportsInterface(bytes4 interfaceId) public view override(ERC721, IERC165) returns (bool) {
+        return interfaceId == 0x49064906 || super.supportsInterface(interfaceId);
```

Comment:
Someone reading this without context wouldn't understand much. I'll add a comment to explain what `0x49064906` means.

Reply:
Better yet: rewrite it as `type(IERC4906).interfaceId`

Reply:
Good point.